tdrs-frontend/package.json

```diff
@@ -70,36 +62,24 @@
     ]
   },
   "devDependencies": {
-    "@testing-library/dom": "^7.21.5",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
-    "@testing-library/user-event": "^7.1.2",
```

Comment:
`@testing-library/user-event`

I think we should keep this as I'm actually using it in one of my branches